server/sonar-web/src/main/js/apps/settings/components/DefinitionDefaults.js

```diff
@@ -19,21 +19,71 @@
  */
 // @flow
 import React from 'react';
-import PropTypes from 'prop-types';
+import Modal from 'react-modal';
 import { getSettingValue, isEmptyValue, getDefaultValue } from '../utils';
 import { translate } from '../../../helpers/l10n';
 
+/*::
+type Props = {
+  isDefault: boolean,
+  onReset: () => void,
+  setting: Object
+};
+*/
+/*::
+type State = { reseting: boolean };
+*/
+
 export default class DefinitionDefaults extends React.PureComponent {
-  static propTypes = {
-    setting: PropTypes.object.isRequired,
-    isDefault: PropTypes.bool.isRequired,
-    onReset: PropTypes.func.isRequired
+  /*:: props: Props; */
+  state /*: State*/ = { reseting: false };
+
+  handleClose = () => {
+    this.setState({ reseting: false });
   };
 
-  handleReset(e /*: Object */) {
+  handleReset = (e /*: Event & {target: HTMLElement} */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)